incomplete-cat-B-items.md

```diff
@@ -0,0 +1,116 @@
+This file has been auto-generated based on README.md. Do not edit it manually
+
+| Cat | Status   | Reference | Category                        | Description                                             | Comment
+| --- | -------- | --------- | ------------------------------- | ------------------------------------------------------- | -------
+|  B+ |          | 2.4.2     | theme                           | `titleLayout`                                           |
+|  B  |          | 1.2.6     | wayland-protocol                | input-method-unstable-v1                                |
+|  B  |          | 1.2.7     | wayland-protocol                | input-timestamps-unstable-v1                            |
+|  B  |          | 1.2.8     | wayland-protocol                | keyboard-shortcuts-inhibit-unstable-v1                  |
+|  B  |          | 1.2.9     | wayland-protocol                | linux-dmabuf-unstable-v1                                |
+|  B  |          | 1.2.10    | wayland-protocol                | linux-explicit-synchronization-unstable-v1              |
+|  B  |          | 1.2.12    | wayland-protocol                | pointer-gestures-unstable-v1                            |
+|  B  |          | 1.2.15    | wayland-protocol                | tablet-unstable-v2                                      |
+|  B  |          | 1.2.16    | wayland-protocol                | text-input-unstable-v3                                  |
+|  B  |          | 1.2.18    | wayland-protocol                | xdg-foreign-unstable-v2                                 |
+|  B  |          | 1.2.20    | wayland-protocol                | xwayland-keyboard-grab-unstable-v1                      |
+|  B  |          | 1.2.21    | wayland-protocol                | drm-lease-v1                                            |
```

Comment:
I think `drm_lease` was [implemented](https://github.com/labwc/labwc/pulls?q=lease) some time ago. I am not convinced if its implemented properly regarding giving back the lease though.

Reply:
The footnote I did for `ext-session-lock-v1` seems a nice way of adding long comments/paragraphs.

Reply:
Well, I didn't dig into the protocol implementation in wlroots, so maybe it actually is implemented correctly and returning the lease is handled internally in wlroots and we just get a new output notification on our side. Just thought I should mention this the chance of it not being implemented completely.

We could mark the protocol as implemented and if there are bugs around it we'll fix them anyway. unrelated to the scope document.